.travis.yml

```diff
@@ -30,6 +30,9 @@ before_script:
 - sudo scripts/bootstrap.sh $ENVIRONMENT $BASE_FOLDER
 - openssl req -x509 -newkey rsa:4096 -nodes -out web_s2i/cert.pem -keyout web_s2i/key.pem -days 365 -subj "/C=IL/ST=Gush-Dan/L=Tel-Aviv/O=DevOps Loft/OU=''/CN=''"
 - scripts/docker-compose-provision.sh $ENVIRONMENT $BASE_FOLDER $WEB_HOST_PORT $WEB_GUEST_PORT $WEB_HOST_SECURE_PORT $WEB_GUEST_SECURE_PORT
+- docker build -t spinner .
+- docker run -t -d --name spincontainer -v /var/run/docker.sock:/var/run/docker.sock spinner
+- docker exec -it spincontainer ecs-cli --version
```

Comment:
this is the test. It should be in the script section

Reply:
So is it ok to leave the build and run at before script section and just move the docker exec to the script section?

Reply:
This is one option.
Second option is implementing a testing script (bash or python). Execute it in the script section.
Third option is implementing a pytest test.

Reply:
Do not forget next time to comment here before requesting review :-)